src/store/RootState.ts

```diff
@@ -13,4 +13,8 @@ export interface ConditionRow {
 	valueData: {
 		value: string;
 	};
+	// TODO: determine what the name should be
+	propertyValueRelationData: {
+		value: string;
```

Comment:
As soon as #67 is merged, we could use that enum for this instead of string literals

Reply:
Yes, agree